### PR TITLE
Update cryptofeed integration

### PR DESCRIPTION
**Pull request by Herklos**
Cryptofeed is preparing 2.0.0 update release

### CODE REVIEW

File: octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py [hunk 1]
```diff
@@ -508,10 +507,10 @@ def _convert_book_prices_to_orders(self, book_prices, book_side):
         return [
             {
                 ECOBIC.PRICE.value: float(order_price),
-                ECOBIC.SIZE.value: float(order_size),
+                ECOBIC.SIZE.value: float(book_prices[order_price]),
                 ECOBIC.SIDE.value: book_side,
             }
-            for order_price, order_size in book_prices.items()
+            for order_price in book_prices
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:513]: I don't understand why this change ?

File: octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py [hunk 2]
```diff
@@ -559,159 +558,183 @@ def _fix_logger(self):
...
-    async def ticker(self, feed, symbol, bid, ask, timestamp, receipt_timestamp):
-        if symbol:
-            symbol = self.get_pair_from_exchange(symbol)
-            # Can't create a full ticker from bid, ask, timestamp and symbol data
-            # Push (ask + bid) / 2 as close price in MARK_PRICE channel
-            await self.push_to_channel(channel_name=trading_constants.MARK_PRICE_CHANNEL,
-                                       symbol=symbol,
-                                       mark_price=float((ask + bid) / 2),
-                                       mark_price_source=trading_enums.MarkPriceSources.TICKER_CLOSE_PRICE.value)
+    async def ticker(self, ticker, receipt_timestamp):
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:561]: should we add typing in args ?

**Review by GuillaumeDSM — COMMENTED** [review]
really nice this 2.0 version !

File: octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py (hunk 1 above)
Comment (Herklos) [reply octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:513]: Good question, I don't remember why, I'll recheck this

File: octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py (hunk 2 above)
Comment (Herklos) [reply octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:561]: Yes we can

Comment (GuillaumeDSM) [reply octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:561]: I think that would be great !

**Comment by Herklos** [comment]
Thanks for your review @GuillaumeDSM, it's done :)
